wrappers/Python/CoolProp/Plots/HSFlashTimingMap.py

```
@@ -0,0 +1,231 @@
+
+    Returns a dict with the p-T saturation curve and the h-s two-phase dome
+    (liquid branch up + vapor branch back down, a closed outline) plus the
+    critical point in each system.  Returns {} if CoolProp is unavailable."""
+    try:
+        import CoolProp.CoolProp as CP
+    except ImportError:
+        return {}
+    try:
+        Tt = CP.PropsSI("Ttriple", fluid)
+        Tc = CP.PropsSI("Tcrit", fluid)
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)